docs/test_plan/proctored_exam.rst

```diff
@@ -0,0 +1,133 @@
+
+
+Proctorio-Specific Tests
+========================
+These tests are for those using Proctorio as their proctoring provider
+
+Exam Flow Works as Expected
+---------------------------
+#. With Proctorio not installed, clicking "Start System Check" redirects you to https://getproctorio.com/
+#. Go through the Proctorio Setup for your Webcam, Desktop, etc. Upon setup completion, your webcam should appear to the right of the screen along with some controls. 
+#. On the "Return to exam" page, DO NOT click on the "Sending you back to your exam" page. Wait instead, and you should be sent to the "Important"/"Start Exam" interstitial automatically
+#. In a separate exam, go through the setup again and on the "Sending you back to your exam." page, click "Return to Exam". Clicking this link should send you to another interstitial. (NOTE: Clicking this link is time-bound so make sure to click it fast)
+#. Clicking the "Return to Exam" button should direct you to a page titled "Important" w/ another header saying "Proctored Exam Rules", and a "Start Exam" Button
+
+
+Exams Dashboard
+===============
+Expected Behavior: Calls to the ACS Endpoint work, instructors can review/reset exam attempts, and expected downstream effects (e.g. w/ completion, credits, certificates, etc) of modifying an exam attempt's status occur
+NOTE: These instructions currently do not account for downstream effects involving a learner's credit requirement status. We currently have no plan to include these
+
+Dashboard UI Loads Exam Attempts
+--------------------------------
+#. Clicking on the "Special Exams" tab should load the *new* version of the exams dashboard
+#. Clicking on the "Review Dashboard" tab within the exams dashboard should open a dashboard owned by your selected procotoring software (e.g. a Proctortrack or Proctorio dashboard for reviewing exam attempt details)
+#. Clicking the "Select An Exam" dropdown should show the list of exams for the course
+#. Filtering by exam name in the exam selection dropdown should bring up the expected exams (e.g. querying "M4-M6" shows all the exams in that section)
+#. Selecting a specific exam (i.e. the ones you just took) should load the respective exam attempts for those exams
+#. The exam attempts should be in the expected status, e.g. successful attempts are submitted or verified, and the errored attempts are errored
+#. Filtering by username only should show exam attempts for that username
+#. Clicking the "Review Dashboard" link in the modal should load the review dashboard for your chosen Proctoring Software (e.g. Proctortrack or Proctorio)
+
+Can Reject/Verify "Review Required" attempts
+---------------------------------------------------
+INSTRUCTIONS: Go into another exam and be suspiscious. Block your webcam, leave for a few seconds, make strange noises, open a bunch of tabs, watch some youtube, open wikipedia, ask chatgpt for the answer to life, etc. Note you will need to do this a couple of times
```

Comment:
Out of curiosity, how does this work? Does Proctorio return a response this quickly? I know Proctortrack would take some time to return back a review.

Reply:
Usually takes no more than a minute from what I've seen on Stage.

Reply:
Oh, nice!